auth-web/src/routes/router.ts

```diff
@@ -222,6 +224,14 @@ export function getRoutes (): RouteConfig[] {
           path: 'product-settings',
           name: 'product-settings',
           component: productPackage
+        },
+        {
+          path: 'activity-log',
+          name: 'activity-log',
+          component: activityLog,
+          meta: {
+            isPremiumOnly: true
```

Comment:
Don't we need to show activity logs to staff for all (including basic) accounts ?